src/components/Orders.js

```diff
@@ -1,6 +1,6 @@
 import React from "react";
 import "./Orders.css";
-import DataTable from "./DataTable";
+import OrderDataTable from "./OrderDataTable";
```

Comment:
If we create a new Component for InventoryTable as well then you should probably remove DataTable